Formula/glibc.rb

```diff
@@ -190,9 +210,95 @@ def post_install
     share.install_symlink sys_zoneinfo if sys_zoneinfo.exist? && !brew_zoneinfo.exist?
   end
 
+  def caveats
+    <<~EOS
+      The Homebrew's Glibc has been installed with the following executables:
+        #{opt_bin}/ldd
+        #{opt_bin}/ld.so
+        #{opt_sbin}/ldconfig
+
+      By default, Homebrew's linker will not search for the system's libraries. If you
+      want Homebrew to do so, run:
+
+        cp "#{HOMEBREW_PREFIX}/#{system_ld_so_conf}" "#{HOMEBREW_PREFIX}/#{system_ld_so_conf.rpartition(".")[0]}"
+        rm "#{opt_prefix}/etc/ld.so.cache"
+        #{opt_sbin}/ldconfig &>/dev/null
+
+      to append the system libraries to Homebrew's ld search paths. This is risky and
+      **highly not recommended**, because it may cause linkage to Homebrew libraries
+      mixed with system libraries.
+    EOS
+  end
+
   test do
     assert_match "Usage", shell_output("#{lib}/ld-linux-x86-64.so.2 --help")
     safe_system "#{lib}/libc.so.6", "--version"
     safe_system "#{bin}/locale", "--version"
   end
+
+  def ld_so_conf_content
+    <<~EOS
+      # This file is generated by Homebrew. Do not modify.
+      #{lib}  # ensure Homebrew Glibc always comes first
+      include #{etc}/ld.so.conf.d/*.conf
+    EOS
+  end
```

Comment:
This is used only in one place, so defining a method for this seems unnecessary, and creates unnecessary distance between where it is used and where it is defined.